packages/core/src/api/meta/types.ts

```diff
@@ -1,5 +1,7 @@
-import type { MetaUser } from '../../metas';
+import type { MetaSession, MetaUser } from '../../metas';
 
 export interface MetaAPI {
-  setUser: (user: MetaUser | null) => void;
+  setUser: (user?: MetaUser) => void;
```

Comment:
Just a wild suggestion: separate between `setUser` and `resetUser`? As a user of the library it's easier to refactor later (i.e. searching for `resetUser` rather than `setUser(null)` or `setUser(undefined)` etc.

Reply:
But then it will result in code like `if (user) { setUser(user); } else { removeUser() }` which isn't great either :thinking:  I think unless something is going terribly wrong, there should be a single `setUser` or `setSession` in the entire app anyway, so not a big deal to find

Reply:
alright, done, both `resetSession` and `resetUser`